snorkel/labeling/model/label_model.py

```diff
@@ -87,7 +87,22 @@ class _CliqueData(NamedTuple):
 
 
 class LabelModel(nn.Module):
-    """A conditionally independent LabelModel to learn LF weights and assign training labels.
+    r"""A model for learning the LF accuracies and combining their output labels.
+
+    This class estimates a model of the labeling functions' conditional probabilities
```

Comment:
estimates a model -> learns a model?
correctly outputting the true -> outputting the true